src/Animated.js

```diff
@@ -48,12 +33,13 @@ const Animated = {
   ...derived,
 
   // animations
-  decay: backwardsCompatibleAnim(decay, DecayAnimation),
-  timing: backwardsCompatibleAnim(timing, TimingAnimation),
-  spring: backwardsCompatibleAnim(spring, SpringAnimation),
+  decay: backwardsCompatibleWrapper(decay, DecayAnimation),
```

Comment:
can we rename this so that it contains "Anim" or sth that would indicate it relates to animation